infra/cifuzz/clusterfuzz_deployment.py

```diff
@@ -109,8 +109,10 @@ def download_latest_build(self):
     build_name = self._get_build_name()
 
     try:
+      logging.info('ClusterFuzzLite: downloading latest build.')
```

Comment:
why do you need "ClusterFuzzLite:" in all these messages ? We might change name or something, bad to hardcode product name in error messages.

Reply:
I removed it, but I'm not sure I agree. This isn't the product name, it's the name of the class implementing BaseClusterFuzzDeployment. It lets the reader distinguish between the OSSFuzz implementation and ClusterFuzzLite implementation.